compiler/src/iree/compiler/Codegen/LLVMCPU/test/vector_transpose_lowering.mlir

```diff
@@ -0,0 +1,16 @@
+// RUN: iree-opt --iree-codegen-llvmcpu-vector-transpose-lowering-pipeline --split-input-file %s | FileCheck %s
```

Comment:
should the pass be `iree-llvmcpu-vector-transpose-lowering`?

Reply:
Interesting enough, I didn't see the error locally... Sorry about that